Client/Frontend/Browser/SimpleToast.swift

```diff
@@ -69,6 +94,12 @@ struct SimpleToast {
         stack.addArrangedSubview(rightSpace)
         return stack
     }
+    
+    func dismiss() {
+        DispatchQueue.main.async {
+            self.toastView.map(self.dismiss)
```

Comment:
It took me some time understanding this logic as I haven't seen that there's a second method called `dismiss` but with the view parameter 🤔 What do you think of making the difference more opaque by renaming one of the methods?